### DISCUSSION TITLE
Older version for windows 8.1 ?

### DISCUSSION THREAD

**Discussion by ArmanAksoy** [ccbogel/QualCoder#923] [General]
Hello everyone,

Would anyone have an older version of the qualcoder exe ?

I have to use my old laptop as I'll be away from home. The laptop is running on windows 8.1 and it is so old that I cannot upgrade it to windows 10 or above. I desperately need an older version of Qualcoder that can run on windows 8.1 as the current version is giving me the error below:

"Failed to execute script '_main_' due to unhandled exception: DLL load failed while importing QtGui: The specified procedure could not be found"

"Traceback (most recent call last):
File " qualcoder\_main_.py", line 47, in <module>
ImportError: DLL while importing QtGui: The specified procedure could not be found"

Please let me now if you can help me out ! I need to work on my project and finish it while on the road.

Thank you to any kind soul who can help me !

Arman

**Comment by ccbogel** [comment]
There is no older version exe for old Windows versions. You could try installing following the python installation commands for Windows.
